### DISCUSSION TITLE
Welcome to MLB-Model Discussions!

### DISCUSSION THREAD

**Discussion by mitchellz4** [mitchellz4/MLB-Model#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aaaclones** [comment]
Nba is almost over we can give this a shot if you want ??
I have a few ideas in mind if your interested let me know we can fire it up
